src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -71,8 +71,8 @@ public static UserStore getTestInstance(PersistentStorageAgent persistentStorage
   /** This class is a singleton, so its constructor is private. Call getInstance() instead. */
   private UserStore(PersistentStorageAgent persistentStorageAgent) {
     this.persistentStorageAgent = persistentStorageAgent;
+    this.setActivityStore(ActivityStore.getTestInstance(persistentStorageAgent));
```

Comment:
I think that you want to have this in `getTestInstance`, similar to how you have an analogous line in `getInstance`.

Reply:
I can't do that for the UserStore because of this code : `this.addUser("Admin01", "AdminPass01", /*admin=*/ true);`
It requires the activityStore.

Reply:
Hmmm, ok, so then this is related to issue #24, and both probably need to be fixed.  Using `getTestInstance` doesn't seem like the right thing to do here, but to do something reasonable you might need to figure out a new way to add the initial Admin user.  So I'd say it's fine to have this temporary fix for this PR, but leave a TODO and create an Issue and then you can fix it in a follow-up PR.

Reply:
ok,